packages/web/src/components/auth/login.tsx

```diff
@@ -1,11 +1,18 @@
 "use client";
-
+import Image from "next/image";
 import { signIn } from "next-auth/react";
 
 export const Login = () => {
   return (
-    <>
-      <button onClick={() => signIn("github")}>Sign in with GitHub</button>
-    </>
-  );
+    
+    <div className="flex flex-col justify-center items-center p-[250px]">
+     <h1 >Sign in</h1>
+      <button
+        className="p-5 bg-white text-black rounded duration-700 hover:shadow-[0_35px_60px_-15px_rgba(255,255,255,0.7)]"
+        onClick={() => signIn("github")}
+      >
+        <Image src='/icon.png' alt='Github button' width={30} height={30}/>
```

Comment:
Whenever possible, we'll want to use SVGs instead of images. You should be able to do 

```typescript
import { Github } from "lucide-react" 
```

then 

```jsx
<Github />
```